tools/sdk/raydium/helpers.ts

```diff
@@ -0,0 +1,88 @@
+import BigNumber from 'bignumber.js'
+import { BN } from '@project-serum/anchor'
+import {
+  jsonInfo2PoolKeys,
+  Liquidity,
+  LiquidityPoolKeys,
+  Percent,
+  Token,
+  TokenAmount,
+} from '@raydium-io/raydium-sdk'
+import { PublicKey } from '@solana/web3.js'
+import { findATAAddrSync } from '@utils/ataTools'
+import { ConnectionContext } from '@utils/connection'
+import { liquidityPoolKeysList } from './poolKeys'
+
+export const getAmountOut = async (
+  liquidityPool: string,
+  amountIn: number,
+  connection: ConnectionContext,
+  slippage: number //slippage in %
+) => {
+  const poolKeys = getLiquidityPoolKeysByLabel(liquidityPool)
+  const [base, quote] = await Promise.all([
+    connection.current.getTokenSupply(poolKeys.baseMint),
+    connection.current.getTokenSupply(poolKeys.quoteMint),
+  ])
+  const amountInBN = new BN(
+    new BigNumber(Number(amountIn).toFixed(base.value.decimals))
+      .shiftedBy(base.value.decimals)
+      .toString()
+  )
+  const { minAmountOut } = Liquidity.computeCurrencyAmountOut({
+    poolKeys,
+    poolInfo: await Liquidity.fetchInfo({
+      connection: connection.current,
+      poolKeys,
+    }),
+    currencyAmountIn: new TokenAmount(
+      new Token(poolKeys.baseMint, base.value.decimals),
+      amountInBN
+    ),
+    currencyOut: new Token(poolKeys.quoteMint, quote.value.decimals),
+    slippage: new Percent(new BN(slippage), 10), // slippage in 1/1000
+  })
+  const currentPrice = minAmountOut.toFixed(quote.value.decimals)
+
+  return Number(currentPrice) * amountIn
+}
+
+export const getLPMintInfo = async (
+  connection: ConnectionContext,
```

Comment:
should be the Solana/web3 `Connection` object directly rather than context implementation detail object